templates/core/index.html

```diff
@@ -6,1074 +6,382 @@
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/582bee22-avantech-logo.png" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <div class="p-logo-section__item"> <img src="https://assets.ubuntu.com/v1/75502e5c-NXP-Logo.svg" class="p-logo-section__logo" style="max-height:120px; padding:10px" /> </div>
+          <script>
+            let partners_logo = ["https://assets.ubuntu.com/v1/6079d02b-amd-logo.png","https://assets.ubuntu.com/v1/cd765ccc-nvidia-logo.png", "https://assets.ubuntu.com/v1/3cf238fd-Qualcomm-Logo.svg", "https://assets.ubuntu.com/v1/0176f106-rigado-logo.png","https://assets.ubuntu.com/v1/e9dabb1f-adlink+logo.png","https://assets.ubuntu.com/v1/185a14f4-Avnet.png", "https://assets.ubuntu.com/v1/97e29c13-Dell.svg","https://assets.ubuntu.com/v1/156ddbf8-rexroth_logo.svg","https://assets.ubuntu.com/v1/255a3171-abb-logo.svg","https://assets.ubuntu.com/v1/2234031b-Siemens-logo.svg.png","https://assets.ubuntu.com/v1/3ba27c02-osram-logo.svg","https://assets.ubuntu.com/v1/5bf6fd2f-svg+%281%29.svg","https://assets.ubuntu.com/v1/f6a59799-partner-logo-arm.svg","https://assets.ubuntu.com/v1/e4ac6657-xilinx-inc-logo-vector.svg","https://assets.ubuntu.com/v1/0ebb345c-MediaTek_logo.svg"]
+            const logo_section_items = document.querySelector(".p-logo-section__items")
+            let randomIndexArray = []
+            for (i = 1; i <= 5; i+=1) {
+              randomNum = Math.floor(Math.random() * partners_logo.length )
+              var logo = partners_logo[randomNum]
+              if (randomIndexArray.indexOf(randomNum) === -1) {
+                randomIndexArray.push(randomNum)
+                let element = `<div class="p-logo-section__item"> <img src=${logo} class="p-logo-section__logo" style="max-height:120px; padding:10px;" /> </div>`
+                logo_section_items.insertAdjacentHTML('beforeend', element)
+              } else {
+                i--
+              }
+            }
+          </script>
         </div>
-        <h2 class="p-heading--4">Embedded Linux: make or buy?</h2>
-        <p>Commercial support vs. roll-your-own distribution: a dilemma.</p>
-        <p><a href="/engage/embedded-linux-make-or-buy" class="p-button">Read the whitepaper</a></p>
-      </div>
     </div>
   </div>
 </section>
 
 <section class="p-strip">
-  <div class="row u-sv3">
+  <div class="u-fixed-width">
     <h2>For smart use cases</h2>
-    <p class="p-heading--4">Get Core. Get snaps. Get going.</p>
+    <h3 class="u-sv3">Get Core. Get snaps. Get going.</h3>
   </div>
-  <div class="row u-equal-height u-align--center u-vertically-center u-hide--medium u-hide--small">
-    <div class="col-4">
-      <div class="u-sv2">
+  <div class="row u-equal-height u-align--center u-vertically-center">
```

Comment:
The links in this section don't line up with each other:
![image](https://user-images.githubusercontent.com/2376968/173599510-e6865aab-7159-495b-987b-785d5be7aa99.png)